host/sgx/calls.c

```diff
@@ -381,11 +381,10 @@ static const char* oe_ocall_str(oe_func_t ocall)
                                        "OE_OCALL_WRITE",
                                        "OE_OCALL_SLEEP",
                                        "OE_OCALL_GET_TIME",
-                                       "OE_OCALL_BACKTRACE_SYMBOLS"
+                                       "OE_OCALL_BACKTRACE_SYMBOLS",
```

Comment:
Since we're cleaning up, what do folks think of leaving out the "OE_OCALL_" prefix? It find it redundant and it makes log messages longer.

Reply:
I'm happy either way, I can also the value in keeping this transparent and greppable.

Reply:
I am for removing them.

---
In reply to: [290203277](https://github.com/microsoft/openenclave/pull/1859#discussion_r290203277) [](ancestors = 290203277)

Reply:
I agree, I'm in favor of removing them.

Reply:
I think we could search based on the pattern "OE_OCALL: <func_name>". So the OE_CALL prefix here is not really needed to make the message greppable.

---
In reply to: [290226050](https://github.com/microsoft/openenclave/pull/1859#discussion_r290226050) [](ancestors = 290226050)

Reply:
Ok, I'll take them out then.